test/recipes/70-test_key_share.t

```diff
@@ -290,7 +290,7 @@ $proxy->start();
 ok(TLSProxy::Message->fail(), "Server sends HRR with no key_shares");
 
 SKIP: {
-    skip "No EC support in this OpenSSL build", 1 if disabled("ec");
+    skip "No EC or EDWARD support in this OpenSSL build", 1 if disabled("ec") || disabled("ecx");
```

Comment:
IMO this test should pass with no-ecx

Reply:
Even I removed X25519 in this case (L297), it still fails. any ideas about it?
54540000:error:0A000417:SSL routines:ssl3_read_bytes:ssl/tls alert illegal parameter:..\openssl\ssl\record\rec_layer_s3.c:855:SSL alert number 47
   0 items in the session cache
   0 client connects (SSL_connect())
   0 client renegotiates (SSL_connect())
   0 client connects that finished
   1 server accepts (SSL_accept())
   0 server renegotiates (SSL_accept())
   0 server accepts that finished
   0 session cache hits
   0 session cache misses
   0 session cache timeouts
   0 callback cache hits
   0 cache full overflows (20480 allowed)
Waiting for s_server process to close: 23116...
Waiting for s_client process to close: 38796...
not ok 23 - Client sends a key_share for a Non TLSv1.3 group